isotpsniffer.c

```diff
@@ -373,16 +376,16 @@ int main(int argc, char **argv)
 		}
 
 		if (FD_ISSET(s, &rdfs)) {
-			nbytes = read(s, buffer, 4096);
+			nbytes = read(s, buffer, PDU_BUF_SIZE);
 			if (nbytes < 0) {
 				perror("read socket s");
 				r = 1;
-				if(!noquit)
+				if(!ignore_errors)
 					goto out;
 			}
-			if (nbytes > 4095) {
+			if (nbytes > (PDU_BUF_SIZE - 1)) {
 				r = 1;
-				perror("read socket s too much data");
+				fprintf(stderr, "PDU length %d longer than PDU buffer: %s\n", nbytes, strerror(errno));
```

Comment:
fprintf(stderr, "PDU length %d longer than PDU buffer size %d\n", nbytes, PDU_BUF_SIZE - 1);

(same applies for line 406)

Additionally please set PDU_BUF_SIZE to 8300 as suggested in my previous review.

Can you finally please put all these things into one single patch for the PR.
If this turns out to be too complicated, feel free to create a new PR.

Many thanks!